worker.js

```diff
@@ -231,7 +240,17 @@ function click(selector) {
         var message = '';
         var elements = document.querySelectorAll(selector);
         if (elements !== null && elements.length === 1) {
-            elements[0].click();
+            // TODO: click on an <a> element is not supported in all browsers by default.
+            // find a more clean solution.
+            if (typeof elements[0].click === 'function') {
+                elements[0].click();
+            } else if (elements[0].fireEvent) {
+                elements[0].fireEvent('on' + 'click');
```

Comment:
Why do you concatenate two strings here? 'on' + 'click'
Can you put them together